Build failing on CI, on Linux

Currently, libzmq CI build is [failing](https://travis-ci.org/zeromq/libzmq/jobs/103594213).

I'm also getting the same error in czmq-ffi-gen: [here](https://travis-ci.org/paddor/czmq-ffi-gen/jobs/103648750).

```
[...]
  CXX    src/src_libzmq_la-vmci.lo
  CXX    src/src_libzmq_la-vmci_address.lo
  CXX    src/src_libzmq_la-vmci_connecter.lo
  CXX    src/src_libzmq_la-vmci_listener.lo
  CXX    src/src_libzmq_la-xpub.lo
  CXX    src/src_libzmq_la-xsub.lo
  CXX    src/src_libzmq_la-zmq.lo
src/zmq.cpp: In function ‘int s_sendmsg(zmq::socket_base_t*, zmq_msg_t*, int)’:
src/zmq.cpp:354:24: error: ‘INT_MAX’ was not declared in this scope
src/zmq.cpp: In function ‘int s_recvmsg(zmq::socket_base_t*, zmq_msg_t*, int)’:
src/zmq.cpp:466:24: error: ‘INT_MAX’ was not declared in this scope
src/zmq.cpp:467:1: error: control reaches end of non-void function [-Werror=return-type]
src/zmq.cpp: In function ‘int s_sendmsg(zmq::socket_base_t*, zmq_msg_t*, int)’:
src/zmq.cpp:355:1: error: control reaches end of non-void function [-Werror=return-type]
cc1plus: all warnings being treated as errors
make[1]: *** [src/src_libzmq_la-zmq.lo] Error 1
make[1]: Leaving directory `/home/travis/src/libzmq'
make: *** [all-recursive] Error 1
```

--- a/src/zmq.cpp
+++ b/src/zmq.cpp
@@ -63,6 +63,7 @@ struct iovec {
 #include <string.h>
 #include <stdlib.h>
 #include <new>
+#include <climits>
 
 #include "proxy.hpp"
 #include "socket_base.hpp"

--- a/tests/test_large_msg.cpp
+++ b/tests/test_large_msg.cpp
@@ -28,6 +28,7 @@
 */
 
 #include <limits.h>
+#include <stdint.h>
 #include "testutil.hpp"
 
 int main (void)

--- a/tests/test_large_msg.cpp
+++ b/tests/test_large_msg.cpp
@@ -28,7 +28,6 @@
 */
 
 #include <limits.h>
-#include <stdint.h>
 #include "testutil.hpp"
 
 int main (void)
@@ -49,7 +48,6 @@ int main (void)
     
     zmq_msg_t msg, rcv;
     size_t big = 64 + (size_t) INT_MAX;
-    printf("Large msg: %u %llu %lu\n", INT_MAX, SIZE_MAX, big);
     
     rc = zmq_msg_init_size (&msg, big);
     assert (rc == 0);

